common/lib/xmodule/xmodule/js/main_requirejs.js

```diff
@@ -0,0 +1,8 @@
+(function(requirejs) {
+    requirejs.config({
+        paths: {
+            "moment": "xmodule/include/common_static/js/vendor/moment.min"
```

Comment:
It seems odd to have this file with just one entry in the path mapping. Is this really the only thing that needs to be configured here? Is there a different configuration file where this could be added, instead of creating this new file with one entry?